frontend/src/pages/login.tsx

```diff
@@ -181,6 +196,19 @@ const Login = () => {
           </div>
         )}
       </Form.Group>
+
+      {isRegistering && (
+        <Form.Group className="mb-5" controlId="confirm-password">
+          <Form.Label>Confirm Password</Form.Label>
+          <Form.Control 
+            type="password"
+            placeholder="Confirm Password"
+            onBlur={e => setConfirmPassword(e.target.value)}
+            autoComplete="current-password"
+          />
+        </Form.Group>
+      )}
+
```

Comment:
do we have a way to confirm that passwords match? if mismatch, we should show an error message

Reply:
Yes, we do. It should be in userLogin.ts

Reply:
can you link the code pointer?

Reply:
Pinged you in another comment.